docs/api_reference/src/quantities.tex

```diff
@@ -2812,6 +3101,723 @@
+\tcode{mp_units::one} otherwise.
+
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+return mp_units::quantity{@\placeholdernc{TO-NUMERICAL-VALUE}@(lhs) @\atsign@ @\placeholdernc{TO-NUMERICAL-VALUE}@(rhs),
+                          @\placeholdernc{REFERENCE-OF}@(lhs) @\atsign@ @\placeholdernc{REFERENCE-OF}@(rhs)};
+\end{codeblock}
+\end{itemdescr}
+
+\rSec3[qty.cmp]{Comparison}
+
+In the following descriptions,
+let \tcode{@} be the \fakegrammarterm{operator}, and
+let \placeholder{C} be
+\tcode{std::\stdconcept{equality_comparable}} if \tcode{@} is \tcode{==}, and
+\tcode{std::\stdconcept{three_way_comparable}} if \tcode{@} is \tcode{<=>}.
+
+\indexlibrarymember{quantity}{operator==}
+\indexlibrarymember{quantity}{operator<=>}
+\begin{itemdecl}
+template<std::@\stdconcept{derived_from}@<quantity> Q, auto R2, typename Rep2>
+  requires @\seebelownc@
+friend constexpr bool operator==(const Q& lhs, const quantity<R2, Rep2>& rhs);
+template<std::@\stdconcept{derived_from}@<quantity> Q, auto R2, typename Rep2>
+  requires @\seebelownc@
+friend constexpr bool operator<=>(const Q& lhs, const quantity<R2, Rep2>& rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+using ct = std::common_type_t<quantity, quantity<R2, Rep2>>;
+const ct ct_lhs(lhs);
+const ct ct_rhs(rhs);
+return ct_lhs.numerical_value_ref_in(ct::unit) @\atsign@ ct_rhs.numerical_value_ref_in(ct::unit);
+\end{codeblock}
+
+\pnum
+\remarks
+The expression in the \fakegrammarterm{requires-clause} is equivalent to:
+\begin{codeblock}
+requires requires {
+  typename std::common_type_t<quantity, quantity<R2, Rep2>>;
+} && @\placeholder{C}@<typename std::common_type_t<quantity, quantity<R2, Rep2>>::rep>
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{quantity}{operator==}
+\indexlibrarymember{quantity}{operator<=>}
+\begin{itemdecl}
+template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+  requires @\seebelownc@
+friend constexpr bool operator==(const Q& lhs, const Value& rhs);
+template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+  requires @\seebelownc@
+friend constexpr bool operator<=>(const Q& lhs, const Value& rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{lhs.numerical_value_ref_in(unit) @ rhs}.
+
+\pnum
+\remarks
+The expression in the \fakegrammarterm{requires-clause} is equivalent to:
+\begin{codeblock}
+(Q::unit == ::mp_units::one) && @\placeholder{C}@<Rep, Value>
+\end{codeblock}
+\end{itemdescr}
+
+\rSec3[qty.non.mem.conv]{Non-member conversions}
+
+\begin{itemdecl}
+template<@\libconcept{Quantity}@ To, typename FwdFrom, @\libconcept{Quantity}@ From = std::remove_cvref_t<FwdFrom>>
+  requires @\seebelownc@
+constexpr To @\exposidnc{sudo-cast}@(FwdFrom&& q);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+TBD.
+\end{itemdescr}
+
```

Comment:
We probably also need `value_cast` and `quantity_cast` here.

Reply:
I missed `quantity_cast`.
I guided myself by following the `#include`s of `quantity.h`.
Is there anything else missing?
`value_cast` was in the uploaded documentations, I just missed amending the commit.

Reply:
I think that we are still missing:
- text output and symbol_text
- compare
- quantity_point and construction helpers

Those can be added later:
- system_reference
- math
- random

Maybe we should just extract `pow<>`, `sqrt<>`, and `cbrt<>` from math and put them in quantity.h? The problem is that they may not be `constexpr` on some compilers.